### ISSUE TITLE
[Bug-App]: Relative Time on X-axis Not Showing Meaningful Units

### ISSUE THREAD

**Issue by tylerlum** [wandb/wandb#8700]
### Describe the bug

When I use "Relative Time (Wall)" or "Relative Time (Process)", it gives me no units (is about 40k for 12 hours, so probably in seconds). It used to show it in minutes or hours dynamically. This makes it very hard to understand the time axis and not nice to share updates with my labmates.

Current behavior: 

![Image](https://github.com/user-attachments/assets/3622d5df-6686-41d6-b8d1-b3f35e88db54)

Current behavior on System section (defaults to x-axis being time)

![Image](https://github.com/user-attachments/assets/7ff2bdc8-530c-4a81-b11b-ad8976195fc2)

Old Behavior on System section from video (https://www.youtube.com/watch?v=gnD8BFuyVUA)

![Image](https://github.com/user-attachments/assets/0f0e964b-c35f-43ce-a7e4-fe6fedcf6412)

Browser and OS:
* Google Chrome: Same issue on Ubuntu, Android, and Windows

**Comment by paulosabile-wb** [comment]
Hi @tylerlum Thank you for reporting this to us as well. May I know your current SDK version when you logged this experiments to wandb so I can try to reproduce the issue and investigate? Thank you!

**Comment by tylerlum** [comment]
```
python 
Python 3.8.20 | packaged by conda-forge | (default, Sep 30 2024, 17:52:49) 
[GCC 13.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import wandb
>>> wandb.__version__
'0.12.21'
```

**Comment by tylerlum** [comment]
I also just tried with this:

```
mamba create -n test_env python=3.10
mamba activate test_env
pip install wandb
```

Then

```
# test_wandb.py

import wandb
import time
import math

# Initialize a new wandb run
wandb.init(project="sine_wave_logging", name="sine_wave_test")

# Set up parameters
num_steps = 100  # Number of steps in the loop
frequency = 0.1  # Frequency of the sine wave

# Log sine wave values in a loop
for step in range(num_steps):
    x = step * frequency
    y = math.sin(x)
    
    # Log to wandb
    wandb.log({"step": step, "x": x, "sine_value": y})
    time.sleep(10)

# Finish the wandb run
wandb.finish()
```

```
python test_wandb.py
```

![Image](https://github.com/user-attachments/assets/cc467fc3-8731-4279-82f6-23b386dbed5c)


^ same problem. The time goes up to 200 (and doesn't show units, but it is seconds), should be turned into minutes by now. 


```
python
Python 3.10.15 | packaged by conda-forge | (main, Oct 16 2024, 01:24:24) [GCC 13.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import wandb
>>> wandb.__version__
'0.18.5'
```

![Image](https://github.com/user-attachments/assets/bb88acb8-54e7-4c37-b8d1-3a72fac792bb)

This is the latest version and still has the problem

**Comment by tylerlum** [comment]
![Image](https://github.com/user-attachments/assets/d930e938-5331-4389-b114-8405bcf1103a)

Been 350 seconds, still not showing it in minutes

**Comment by paulosabile-wb** [comment]
Thanks for sharing these useful information @tylerlum I will further investigate this and will get back to you for another update tomorrow. Along with the another issue we are checking with downloading PNG. Thank you!

**Comment by tylerlum** [comment]
Thank you! Any update on this?

**Comment by tylerlum** [comment]
Any update on this?

**Comment by tylerlum** [comment]
Yes, it used to change dynamically so it would show in minutes or hours (whichever was appropriate). However, now everything shows in seconds. The video tutorial is what the behavior used to be like for me before (which I liked a lot). 

Now, if I train for 1.5 days, I would like it to tell me 1.5 days or 36 hours, but it actually shows me 129600 seconds, which is very hard to interpret.

**Comment by tylerlum** [comment]
Thank you for the update! This will be very helpful when fixed

**Comment by tylerlum** [comment]
Yes that workaround works for me. This would still be great to fix, but this workaround is helpful, thanks!

**Comment by tylerlum** [comment]
![Image](https://github.com/user-attachments/assets/6552fbb5-f062-4222-a0d6-28c7efe81a87)

Yes, it looks fixed to me! This is very very helpful, thank you so much!
